### PR TITLE
ticdc: add `required-acks` configuration

### PR DESCRIPTION
**Pull request by hi-rustin**
Signed-off-by: hi-rustin <rustin.liu@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

ref https://github.com/pingcap/tiflow/pull/8154

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: ticdc/ticdc-sink-to-kafka.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ URI 中可配置的的参数如下：
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）。|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值分别从 64MB 和 256MB 调整至 10 MB。|
 | `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）。                       |
+| `required-acks`      | 在 Produce 请求中使用的配置项，用于告诉 broker 需要收到多少副本确认后才进行响应。以下三个常量值都是有效的：0（NoResponse: 不发送任何响应，只有 TCP ACK），1（WaitForLocal: 仅等待本地提交成功后再响应） 和 -1（WaitForAll: 等待所有同步副本提交后再响应。要配置的最小同步副本数量通过 broker 的 min.insync.replicas 配置项进行配置）。                      |
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-kafka.md:59]:
```suggestion
| `required-acks`      | 在 `Produce` 请求中使用的配置项，用于告诉 broker 需要收到多少副本确认后才进行响应。可选值有 ：`0`（`NoResponse`：不发送任何响应，只有 TCP ACK）、`1`（`WaitForLocal`：等待本地提交成功后再响应） 和 `-1`（`WaitForAll`：等待所有同步副本提交后再响应。最小同步副本数量通过 broker 的 `min.insync.replicas` 配置项进行配置）。                      |
```

Comment (shichun-0415) [reply ticdc/ticdc-sink-to-kafka.md:59]: 有关于 min.insync.replicas 配置项的介绍吗（我没搜到）？建议贴上这个配置项的链接。

File: ticdc/ticdc-sink-to-kafka.md [hunk 2]
```diff
@@ -56,6 +56,7 @@ URI 中可配置的的参数如下：
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）。|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值分别从 64MB 和 256MB 调整至 10 MB。|
 | `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）。                       |
+| `required-acks`      | 在 Produce 请求中使用的配置项，用于告诉 broker 需要收到多少副本确认后才进行响应。以下三个常量值都是有效的：0（NoResponse: 不发送任何响应，只有 TCP ACK），1（WaitForLocal: 仅等待本地提交成功后再响应） 和 -1（WaitForAll: 等待所有同步副本提交后再响应。要配置的最小同步副本数量通过 broker 的 [min.insync.replicas](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置）。                      |
```
Comment (sdojjy) [inline ticdc/ticdc-sink-to-kafka.md:59]: 需要加上是否可选以及默认值

**Review by sdojjy — APPROVED** [review]

File: ticdc/ticdc-sink-to-kafka.md [hunk 3]
```diff
@@ -56,6 +56,7 @@ URI 中可配置的的参数如下：
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）。|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值分别从 64MB 和 256MB 调整至 10 MB。|
 | `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）。                       |
+| `required-acks`      | 在 `Produce` 请求中使用的配置项，用于告诉 broker 需要收到多少副本确认后才进行响应。以下三个常量值都是有效的：`0`（`NoResponse`: 不发送任何响应，只有 TCP ACK），`1`（`WaitForLocal`: 仅等待本地提交成功后再响应） 和 `-1`（`WaitForAll`: 等待所有同步副本提交后再响应。要配置的最小同步副本数量通过 broker 的 [`min.insync.replicas`](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置）。（可选，默认值 `-1`）。                      |
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-kafka.md:59]:
```suggestion
| `required-acks`      | 在 `Produce` 请求中使用的配置项，用于告诉 broker 需要收到多少副本确认后才进行响应。以下三个常量值都是有效的：`0`（`NoResponse`：不发送任何响应，只有 TCP ACK），`1`（`WaitForLocal`：仅等待本地提交成功后再响应） 和 `-1`（`WaitForAll`：等待所有同步副本提交后再响应。要配置的最小同步副本数量通过 broker 的 [`min.insync.replicas`](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置）。（可选，默认值 `-1`）。                      |
```

File: ticdc/ticdc-sink-to-kafka.md [hunk 4]
```diff
@@ -56,6 +56,7 @@ URI 中可配置的的参数如下：
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）。|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值分别从 64MB 和 256MB 调整至 10 MB。|
 | `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）。                       |
+| `required-acks`      | 在 `Produce` 请求中使用的配置项，用于告诉 broker 需要收到多少副本确认后才进行响应。可选值有：`0`（`NoResponse`：不发送任何响应，只有 TCP ACK），`1`（`WaitForLocal`：仅等待本地提交成功后再响应） 和 `-1`（`WaitForAll`：等待所有同步副本提交后再响应。要配置的最小同步副本数量通过 broker 的 [`min.insync.replicas`](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置）。（可选，默认值 `-1`）。                      |
```
Comment (shichun-0415) [inline ticdc/ticdc-sink-to-kafka.md:59]:
```suggestion
| `required-acks`      | 在 `Produce` 请求中使用的配置项，用于告知 broker 需要收到多少副本确认后才进行响应。可选值有：`0`（`NoResponse`：不发送任何响应，只有 TCP ACK），`1`（`WaitForLocal`：仅等待本地提交成功后再响应） 和 `-1`（`WaitForAll`：等待所有同步副本提交后再响应。最小同步副本数量可通过 broker 的 [`min.insync.replicas`](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置）。（可选，默认值 `-1`）。                      |
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

File: ticdc/ticdc-sink-to-kafka.md [hunk 5]
```diff
@@ -56,6 +56,7 @@ URI 中可配置的的参数如下：
 | `partition-num`      | 下游 Kafka partition 数量（可选，不能大于实际 partition 数量，否则创建同步任务会失败，默认值 `3`）。|
 | `max-message-bytes`  | 每次向 Kafka broker 发送消息的最大数据量（可选，默认值 `10MB`）。从 v5.0.6 和 v4.0.6 开始，默认值分别从 64MB 和 256MB 调整至 10 MB。|
 | `replication-factor` | Kafka 消息保存副本数（可选，默认值 `1`）。                       |
+| `required-acks`      | 在 `Produce` 请求中使用的配置项，用于告知 broker 需要收到多少副本确认后才进行响应。可选值有：`0`（`NoResponse`：不发送任何响应，只有 TCP ACK），`1`（`WaitForLocal`：仅等待本地提交成功后再响应） 和 `-1`（`WaitForAll`：等待所有同步副本提交后再响应。最小同步副本数量可通过 broker 的 [`min.insync.replicas`](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置）。（可选，默认值 `-1`）。                      |
```
Comment (Oreoxmt) [inline ticdc/ticdc-sink-to-kafka.md:59]:
```suggestion
| `required-acks`      | 在 `Produce` 请求中使用的配置项，用于告知 broker 需要收到多少副本确认后才进行响应。可选值有：`0`（`NoResponse`：不发送任何响应，只有 TCP ACK），`1`（`WaitForLocal`：仅等待本地提交成功后再响应）和 `-1`（`WaitForAll`：等待所有同步副本提交后再响应。最小同步副本数量可通过 broker 的 [`min.insync.replicas`](https://kafka.apache.org/33/documentation.html#brokerconfigs_min.insync.replicas) 配置项进行配置）。（可选，默认值为 `-1`）。                      |
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Pull request merged**